packages/vaadin-lumo-styles/src/global/badge.css

```diff
@@ -0,0 +1,162 @@
+[theme~='badge'] {
```

Comment:
Let's also add copyright comment to all files in `global` folder for consistency.

Reply:
Done.